azext_iot/operations/_mqtt.py

```diff
@@ -49,13 +44,26 @@ def message_handler(self, message):
                 message_properties[item] = message_attributes[item]
 
         message_properties.update(message.custom_properties)
-        six.print_("_Message Properties_: {}".format(message_properties))
+
+        output = {
+            "Topic": "/devices/{}/messages/devicebound".format(self.device_id),
+            "Payload": message.data,
```

Comment:
Instead leaving message.data as bytes, it would be good to decode with the given content encoding OR fallback to utf-8...we are doing a similar approach for c2d-message receive in the `dev` branch.